config/deploy/docker/assets/Dockerfile.open-path-cas.base

```diff
@@ -68,8 +68,16 @@ RUN mkdir -p app/assets/stylesheets/theme/styles && \
   rm .env && \
   mkdir tmp/pids
 
+RUN chown -R app:app tmp
+RUN touch /etc/timezone
+RUN ln -sf /app/etc-localtime /etc/localtime
+RUN chown app:app /etc/timezone
+
```

Comment:
I believe this is missing some items.  In the warehouse this is:
```
RUN touch /etc/timezone
RUN ln -sf /app/etc-localtime /etc/localtime
RUN chown -R app:app tmp
RUN chown -R app:app node_modules
RUN chown -R app:app /usr/local/bundle
RUN chown app:app /etc/timezone
```

Reply:
Good catch, I missed those in the diff cause they came after the PR merge. Thanks!